code/datums/components/defibrillator.dm

```diff
@@ -332,6 +340,28 @@
 	busy = FALSE
 	return
 
+/datum/component/defib/proc/combat_fibrillate(mob/user, mob/living/carbon/human/target)
+	if(!istype(target))
+		return
+	busy = TRUE
+	target.adjustStaminaLoss(60)
+	target.emote("gasp")
+	to_chat(target, "<span class='danger'>[user] touches [target] lightly with [parent]!</span>")
+	add_attack_logs(user, target, "Stunned with [parent]")
+	if(!(combat && target.body_position == LYING_DOWN && do_after(user, 1 SECONDS, TRUE, target, TRUE)))
```

Comment:
```suggestion
	if(!(combat && IS_HORIZONTAL(target) && do_after(user, 1 SECONDS, target = target)))
```
not a fan of how the proc is structured honestly, maybe reverse the conditions so there's no extra !( ?

Reply:
I'll see what I can do